mm2src/mm2_main/src/lp_price.rs

```diff
@@ -0,0 +1,350 @@
+use common::mm_number::MmNumber;
+use common::StatusCode;
+use common::{log::{debug, error},
+             mm_number::BigDecimal};
+use mm2_err_handle::prelude::{MmError, OrMmError};
+use mm2_net::transport::SlurpError;
+use std::collections::HashMap;
+use std::str::Utf8Error;
+
+const MY_PRICE_ENDPOINT1: &str = "https://prices.komodo.live:1313/api/v2/tickers";
```

Comment:
Just a question: what do you think about declaring the endpoints as a slice?
Also I think it's worth to rename the constant by removing `MY_` from the beginning 😅 
```rust
const PRICE_ENDPOINDS: [&str; 2] = ["https://prices.komodo.live:1313/api/v2/tickers", "https://prices.cipig.net:1717/api/v2/tickers"];
```

Reply:
**PRICE_ENDPOINTS done